src/data/redux.test.jsx

```diff
@@ -92,15 +94,28 @@ describe('Data layer integration tests', () => {
   });
 
   describe('Test getProctoringSettings', () => {
-    const fetchProctoringSettingsUrl = `${getConfig().LMS_BASE_URL}/api/edx_proctoring/v1/proctored_exam/settings/exam_id/${exam.id}/`;
     const proctoringSettings = Factory.build('proctoringSettings');
 
-    beforeEach(async () => {
-      mergeConfig({ EXAMS_BASE_URL: null });
+    describe('Test legacy URL for getProctoringSettings', () => {
+      beforeEach(async () => {
+        mergeConfig({ EXAMS_BASE_URL: null });
+      });
+
+      it('Should get, and save proctoringSettings', async () => {
+        axiosMock.onGet(fetchExamAttemptsDataLegacyUrl).reply(200, { exam, active_attempt: attempt });
+        axiosMock.onGet(fetchProctoringSettingsLegacyUrl).reply(200, proctoringSettings);
+
+        await executeThunk(thunks.getExamAttemptsData(courseId, contentId), store.dispatch);
+        await executeThunk(thunks.getProctoringSettings(), store.dispatch, store.getState);
+
+        const state = store.getState();
+        expect(state.examState.proctoringSettings).toMatchSnapshot();
+      });
     });
 
     it('Should get, and save proctoringSettings', async () => {
-      axiosMock.onGet(fetchExamAttemptsDataLegacyUrl).reply(200, { exam, active_attempt: attempt });
+      axiosMock.onGet(fetchExamAttemptsDataUrl).replyOnce(200, { exam });
```

Comment:
if you just need to mock out the default exam/attempt call when the UI loads I added a helper function to do this `initWithExamAttempt`